services/contract_call_local.proto

```diff
@@ -132,6 +132,31 @@ message ContractFunctionResult {
      * EVM address described above.)
      */
      google.protobuf.BytesValue evm_address = 9;
+
+    /**
+     * The amount of gas that was used for this call. 
+     *
+     * This field should only be populated when the paired TransactionBody in the record stream is not a
+     * ContractCreateTransactionBody or a ContractCallTransactionBody.     
+     */
+    int64 gas = 10;
+
+    /**
+     * Number of tinybars sent (the function must be payable if this is nonzero).
+     *
+     * This field should only be populated when the paired TransactionBody in the record stream is not a
+     * ContractCreateTransactionBody or a ContractCallTransactionBody.     
+     */
+    int64 amount = 11;
+
+    /**
+     * The parameters passed into the contract call.
+     *
+     * This field should only be populated when the paired TransactionBody in the record stream is not a
+     * ContractCreateTransactionBody or a ContractCallTransactionBody.     
+     */
+    bytes functionParameters = 12;
```

Comment:
Is there a max size on the bytes length that's worthy of note here?

Reply:
No protocol imposed limit.  The functional limit is how much gas you want to pay.

Reply:
Sounds good